examples/step-63/step-63.cc

```diff
@@ -711,35 +742,36 @@ namespace Step63
     // using streamline diffusion, setting $\delta=0$ negates this contribution
     // below and we are left with the standard, Galerkin finite element
     // assembly.
-    const double delta = settings.with_streamline_diffusion ?
-                           compute_stabilization_delta(cell->diameter(),
-                                                       settings.epsilon,
-                                                       advection_direction,
-                                                       settings.fe_degree) :
-                           0.0;
+    const double delta = (settings.with_streamline_diffusion ?
+                            compute_stabilization_delta(cell->diameter(),
+                                                        settings.epsilon,
+                                                        advection_direction,
+                                                        settings.fe_degree) :
+                            0.0);
```

Comment:
What's the reason for the braces?

Reply:
It's just a way for me to avoid having to think about operator precedence questions. We do this quite frequently when using the `?:` operator.

Reply:
Fine with me.